src/main/groovy/io/seqera/wave/service/builder/ContainerBuildServiceImpl.groovy

```diff
@@ -244,6 +248,8 @@ class ContainerBuildServiceImpl implements ContainerBuildService {
     }
 
     protected BuildTrack checkOrSubmit(BuildRequest request) {
+        //increment metrics
+        CompletableFuture.supplyAsync(() -> metricsService.incrementBuildsCounter(request.accessToken), executor)
```

Comment:
Note, the build counter should be incremented just before launching the build job. Look how it was done in the micrometer based implementation 

https://github.com/seqeralabs/wave/commit/c8d55b8712bc52d9d19b5da6077e9bcce0a7fa3a#diff-245709fa24526650302f1dc7da9705d9c24f2ffdd2a11cfd40e62c7483c51cf4R181-R183

Reply:
Ok, noted